### PR TITLE
If you would like to launch sbt with a specific port number directly

### PR DESCRIPTION
**Pull request by kevinobama**
If you would like to launch sbt with a specific port number directly:
```
sbt "run 9005"
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,11 @@ sbt run
 
 And then go to http://localhost:9000 to see the running web application.
 
+If you would like to launch sbt with a specific port number directly:
+```
+sbt "run 9005"
```
Comment (cchantep) [inline README.md:19]: That's part of the Play documentation. I'm for duplicating.

**Review by cchantep — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kevinobama) [reply README.md:19]: where is that? I only found it in stack overflow.

Comment (cchantep) [reply README.md:19]: Anyway if thats not already in the Play main doc, that's there it should be added, not here, as not specific to this example and would be prone to duplication

Comment (kevinobama) [reply README.md:19]: ok.

Comment (marcospereira) [reply README.md:19]: It is documented here:

https://www.playframework.com/documentation/2.6.x/PlayConsole#Using-the-play-commands-directly

I little buried in our docs. I agree with @cchantep that we should make it better there.

**Pull request closed without merging**